src/Confluent.SchemaRegistry/SchemaRegistryConfig.cs

```diff
@@ -80,6 +80,26 @@ public static class PropertyNames
             /// </summary>
             [Obsolete("Subject name strategies should now be configured using the serializer's configuration. In the future, this configuration property will be removed from SchemaRegistryConfig")]
             public const string SchemaRegistryValueSubjectNameStrategy = "schema.registry.value.subject.name.strategy";
+
+            /// <summary>
+            ///     File path to CA certificate(s) for verifying the schema registry's key.
```

Comment:
Need to mention what the default behaviour is (hopefully to use the system CA certs)